explainer.md

```diff
@@ -563,17 +578,34 @@ interface VRLayer {};
 typedef (HTMLCanvasElement or
          OffscreenCanvas) VRCanvasSource;
 
-[Constructor(VRSession session, optional VRCanvasSource source)]
-interface VRCanvasLayer : VRLayer {
-  attribute VRCanvasSource source;
-
-  void setLeftBounds(double left, double bottom, double right, double top);
-  FrozenArray<double> getLeftBounds();
+dictionary VRWebGLLayerInit {
+  boolean antialias = true;
+  boolean depth = false;
+  boolean stencil = false;
+  boolean alpha = true;
+  boolean multiview = false;
+  double bufferScaleFactor;
+};
 
-  void setRightBounds(double left, double bottom, double right, double top);
-  FrozenArray<double> getRightBounds();
+[Constructor(VRSession session, VRCanvasSource source, optional VRWebGLLayerInit layerInit)]
+interface VRWebGLLayer : VRLayer {
+  readonly attribute VRCanvasSource source;
+  readonly attribute boolean antialias;
+  readonly attribute boolean depth;
+  readonly attribute boolean stencil;
+  readonly attribute boolean alpha;
+  readonly attribute boolean multiview;
+  
+  readonly attribute WebGLFrameBuffer frameBuffer;
```

Comment:
I went back and forth on including the framebufferScaleFactor.  Ultimately, I couldn't figure out what developers would use the information for, so I left it off.  If you have a suggestion for how this would be used, I'd be happy to add it in.